spec/fixtures/vcr_cassettes/complexity/get_complexities.yml

```diff
@@ -0,0 +1,107 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://hmpps-complexity-of-need-staging.apps.live-1.cloud-platform.service.justice.gov.uk/v1/complexity-of-need/multiple/offender-no
```

Comment:
The API hostname change presumably means that these VCR cassettes will change too.